source/Irrlicht/COGLES2Driver.cpp

```diff
@@ -2252,7 +2261,7 @@ COGLES2Driver::~COGLES2Driver()
 	void COGLES2Driver::OnResize(const core::dimension2d<u32>& size)
 	{
 		CNullDriver::OnResize(size);
-		CacheHandler->setViewport(0, 0, size.Width, size.Height);
+		setViewPort(core::recti(0, 0, size.Width, size.Height));

```

Comment:
changing `CNullDriver::OnResize(` to set `ViewPort` in all cases appears to work just fine, then you don't need to change the implementation in each driver